engine/components-api/src/main/java/com/cloud/capacity/CapacityManager.java

```diff
@@ -73,16 +73,26 @@
                     "If set to true, creates VMs as full clones on ESX hypervisor",
                     true,
                     ConfigKey.Scope.StoragePool);
-    static final ConfigKey<Integer> ImageStoreNFSVersion =
+    static final ConfigKey<Integer> ImageStoreNFSMajorVersion =
```

Comment:
@Spaceman1984 can you revert the change, why not parse for the major and minor version? Something like `x.y` could be parsed by splitting on the `.` if provided.

Reply:
@rhtyd If I was to split on `.` I would have to change the Integer data type to String, basically undoing all the changes I have made. Doing it this way preserves backward compatibility if a value has already been stored for NFS version and adding a second variable for minor version is as per the suggestion in the original issue.